src/fr/composants/case-dusage.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Same comment here, looks like this was supposed to be part of the checkbox folder.